### ISSUE TITLE
Fix OpenWrt Home Page

### ISSUE THREAD

**Issue by richb-hanover** [prplfoundation/openwrt-doc-initiative#12]
The http://openwrt.org page is not compelling for new (or even experienced) people. It lacks a clear explanation of:
- what OpenWrt is, and that it is a successful project for cool people
- how it could make a difference in your life
- why you will be enormously rewarded if you follow our simple instructions

In addition, most of the links from the home page are grossly outdated (the documentation link is for Kamikaze) or scary (both documentation and download links are plain HTML, and the latter is just a directory listing(!)) or that no one knows about/understands (Battlemesh? I'm sure it's cool, but is it central to OpenWrt?).

The entire page needs rethinking, so that it exudes confidence and welcomes people into the "OpenWrt club" - now that they've found us, their network routers (and their home networking experience) can be so much better (at least reliable, and likely better performing).

**Comment by richb-hanover** [comment]
I will take a crack at revising the content for the OpenWrt home page, and then post a URL to be criticized here by the end of the weekend.

**Comment by wwahammy** [comment]
once again, thanks @richb-hanover. If there's anything I or anyone else can do to help, don't hesitate to ask!

**Comment by richb-hanover** [comment]
**Redesign of the OpenWrt Home Page**

I have revised the content of the OpenWrt home page (http://openwrt.org). It is my intent that the page contain only timely and welcoming information for new people, while providing links to older info, and info for advanced users. The current draft is on my own server at:
http://173.208.152.60/openwrt-redesign/
Comments, please!

[ This note also posted to OpenWrt Forum at https://forum.openwrt.org/viewtopic.php?pid=276602#p276602 ]

**Comment by richb-hanover** [comment]
Chaos Calmer rc1 is out. Consequently, there's an update to the redesigned home page at http://173.208.152.60/openwrt-redesign/ Changes include:
- Minor improvements to the wording in "What is OpenWrt". Careful proofreading is still needed.
- Added CC-rc1 release note above Barrier Breaker
- Minor rearrangement of the side bar (putting Facebook & Flattr below Headlines)
- Summarize Earlier OpenWrt Releases at bottom of page
- Removed "Comments" link from each of the articles, since they frequently go to closed topics on the wiki.
- The github repo now contains all the javascript and CSS files so we can edit them if needed.

**Comment by wwahammy** [comment]
This is a much bigger task but have we considered scrapping the current design totally? It'd be awesome to feature the quality and effectiveness of OpenWrt with a beautiful website. 

I'm not a designer but just skimming through Beautifully Open I think the design for http://beautifulopen.com/2015/03/26/pico/ is great. We could build OpenWrt homepage with a similar design. The links on top could go to the same links we have on top of the OpenWrt homepage now. Below the six features, we could put news stories. Additionally, this could easily be based on Bootstrap thereby making the homepage responsive. 

Any thoughts?

**Comment by richb-hanover** [comment]
There's no doubt that the openwrt.org page could be cleaned up. The Pico design would serve nicely, or the site could even clone the "bootstrap" look of the current LuCI GUI.

But if we're to have success at any of this, we need to get buy-in from the core-developers about permitting these changes. The changes we've proposed so far are about as far as I feel comfortable without any feedback as to their acceptability.

A while ago, I touched base with Gregers Petersen, who asked us for a proposal about changes. Is this good enough to show what we have in mind?

**Comment by zo0ok** [comment]
I opened up the topic (about replacing the wiki):
https://forum.openwrt.org/viewtopic.php?pid=274743#p274743
However, that was more with a focus on the documentation itself, and less on the start page. Using two different platforms did not occur to me by then - and neither later when I confused the wiki start page with the openwrt "homepage". I would not object if the first page or the site looked nicer, of course. But it is important to draw a clear line. And I think it is annoying enough that the forum and the wiki uses separate user/pass... so another platform for the none-wiki-parts of the site, I dont know.

Methinks the navigation structure, and a clear idea of where different types of documentation should go, are the most important issues for the site.

Just a thought that came to me right now... an alternative to improving the start page would be to have a stronger and more focused and designed presence on... twitter? facebook? stackoverflow? github?

That said, I just read richb-hanovers' incoming comment and I agree with him.

A concrete proposal (and perhaps a more long term vision to see where it fits in) would be a good thing to produce.

But as I have mentioned before, I focus my energy on the Table of Hardware... we will eventually start touching the device pages... and there is much work to do.

**Comment by richb-hanover** [comment]
I just updated the prototype OpenWrt Home Page to refer to CC rc2. http://test.richb-hanover.com/openwrt-redesign/ I also pointed all the documentation links to the (new) OpenWrt Guides pages at http://wiki.openwrt.org/doc/playground

Does this group have any further comments on the general layout/contents of the page? 

If not, I plan to touch base with the core developers to ask that the consider switching the home page to use this text. (My suspicion is that the best way to do this would be to file a ticket at https://dev.openwrt.org...) 

Any other thoughts? Thanks.

**Comment by wwahammy** [comment]
@richb-hanover I'm in favor of moving forward with this. I think longer term it'd be great for OpenWrt to have a nice and shiny page but what you've provided is a significant step forward. It'd be good to clarify what process needs to be followed to actually formally propose changes to the main home page in the future as well. So don't hesitate to document how this should be done :)

**Comment by zo0ok** [comment]
I am good with it!

**Comment by richb-hanover** [comment]
Thanks for the support on this. I will submit the ticket and add the URL to it to this ticket.

**Comment by richb-hanover** [comment]
Ticket filed. https://dev.openwrt.org/ticket/19892 

Let's see what kind of response we get...

**Update:** None. No response for three weeks.

**Comment by notorand-it** [comment]
![shot-0465](https://cloud.githubusercontent.com/assets/5338874/13402573/3a7ed422-df11-11e5-9373-3feccc4961b7.png)

One picture is worth more than 1000 words.

**Comment by richb-hanover** [comment]
I agree. The ticket https://dev.openwrt.org/ticket/19892 was posted last June, without response.

**Comment by notorand-it** [comment]
I have a simple and old question: who is in charge for the website?
Not the [wiki](http://wiki.openwrt.org/), not the [bug tracking](https://dev.openwrt.org/), not the [forum](https://forum.openwrt.org/) and not this [GitHub project](https://github.com/prplfoundation/openwrt-doc-initiative).
I mean the stuff that gets out when I point a browser to www.openwrt.org.
Without this tiny piece of information all discussions, suggestions, complaints and the likes are just to be sent to `/dev/null`.

I have a couple of ideas:
1. None.
2. None knows it any more.

**Comment by richb-hanover** [comment]
It's not clear at all. My understanding is that it's the "core developers" who are mostly focussed on enhancing the software. I just updated my ticket at https://dev.openwrt.org/ticket/21942 to ask whether those people would appreciate any help. I'm hopeful that they can hand control to a small team to keep the website/top-of-the-wiki up to date.

**Comment by notorand-it** [comment]
It's completely clear, instead to me.
Two cases:
1. There's none alive any more who can fix it.
2. Those who can fix it are alive and simply don't care.

Easy, isn't it?

**Comment by richb-hanover** [comment]
It's been 24 days since I put in that ticket, so regrettably, I have to conclude that choice number 2 is the answer. Sigh.

**Comment by notorand-it** [comment]
I think it's no.1. Because trac contains tickets to the website that are older than 1 year.

**Comment by richb-hanover** [comment]
I think we have a new opportunity. The OpenWrt home page no longer has outdated info, and mostly has links to the Dev site, the Documentation page, Downloads, the wiki, the forums, and the OpenWrt Facebook page. This is a vast improvement - there's nothing wrong or blatantly outdated there. (It would be nice to have a little explanatory text though.)

In addition, the top-level Documentation page on the wiki is now editable. See my proposal at: https://forum.openwrt.org/viewtopic.php?pid=317553#p317553
